examples/tv-casting-app/linux/main.cpp

```diff
@@ -114,6 +115,30 @@ HelpOptions helpOptions("tv-casting-app", "Usage: tv-casting-app [options]", "1.
 
 OptionSet * allOptions[] = { &cmdLineOptions, &helpOptions, nullptr };
 
+static void OnBindingAdded(const EmberBindingTableEntry & binding)
+{
+    if (binding.type == EMBER_MULTICAST_BINDING)
+    {
+        ChipLogError(NotSpecified, "Group binding received");
+        return;
+    }
+
+    if (binding.type == EMBER_UNICAST_BINDING)
+    {
+        ChipLogError(NotSpecified,
+                     "Unicast binding received nodeId=0x" ChipLogFormatX64 " remote endpoint=%d cluster=" ChipLogFormatMEI,
```

Comment:
Add not implemented data in the error, since just seeing "ERROR: Unicast binding received nodeId=0x... remote endpoint=1 cluster=...." out of context is hard to figure out: why is it an error to receive a unicast binding? 

Something like "Unicast binding handling not implemented (received nodeid.....)"

Reply:
cleaned up